core/tests/protractor_utils/users.js

```diff
@@ -19,15 +19,17 @@
 
 var general = require('./general.js');
 var waitFor = require('./waitFor.js');
-
+var action = require('./action.js');
 var AdminPage = require('./AdminPage.js');
+const { browser } = require('protractor');
 var adminPage = new AdminPage.AdminPage();
 
 var login = async function(
     email, isSuperAdmin = false, manualNavigation = true) {
   // Use of element is not possible because the login page is non-angular.
   // The full url is also necessary.
   var driver = browser.driver;
+  await browser.waitForAngularEnabled(false);
```

Comment:
Why do we need this?

Reply:
If I don't add this, the following test fails:
![Screenshot from 2021-02-13 12-06-50](https://user-images.githubusercontent.com/24855641/107843807-16fc1880-6df4-11eb-91d9-aeb587d661b5.png)
![Screenshot from 2021-02-13 12-07-02](https://user-images.githubusercontent.com/24855641/107843809-195e7280-6df4-11eb-9fa9-62b8561acca5.png)

Reply:
Hi @ashutoshc8101, just a general comment, please don't resolve comments by yourself. It makes it harder for reviewers to see previous comments.

Reply:
Hm, it fails because using action.* expects the page to be an Angular page but the login screen is not Angular currently. Using `waitForAngularEnabled` will disable the waitForAngular check, but it also makes tests flaky so we don't suggest using it.


Here's what I would suggest. Remove `waitForAngularEnabled` here and below that you added and remove usage of "action" in the login / logout screen. I think it should be possible to use action in lines 104-106 without `waitForAngularEnabled` but please confirm this by running any e2e test suite that uses the corresponding method on your local machine.

If it is not possible to use action there, I think we can exclude "users.js" from the list of files that need to use action. Maybe you can add a detailed comment at the top of this file about why we can't use "action".